install.js

```diff
@@ -1,63 +1,89 @@
 'use strict';
 
-const proc = require('child_process');
 const path = require('path');
 const fs = require('fs');
 const os = require('os');
+var get = require('simple-get')
+var pump = require('pump')
+var tfs = require('tar-fs')
+var zlib = require('zlib')
+var pkg = require('./package.json')
+var support = require('./support')
 
-if (process.argv[2] == 'install') {
-  try {
-    require('./index.js');
-  } catch (e) {
-    // Need compile
-    console.log('Recompiling iohook for your environment');
-
-    let currentPlatform = 'iohook_';
-    
-    if (process.versions['electron']) {
-      currentPlatform += 'electron.' + process.versions['electron'] + '_';
-    }
-    
-    currentPlatform += os.platform() + '_' + os.arch();
-    console.log('Platform is:', currentPlatform);
-
-    proc.execSync('git submodule update --init');
-    if (!checkLibuiohookSource()) {
-      deleteFolderRecursive(path.join(__dirname, 'libuiohook'));
-      proc.execSync('git clone https://github.com/kwhat/libuiohook.git');
-    }
-    
-    proc.execSync('npm install nan cmake-js');
-    proc.execSync('npm run compile');
-    // if (fs.existsSync(path.join(__dirname, 'build', 'Release', 'iohook.node'))) {
-    //   fs.renameSync(
-    //     path.join(__dirname, 'build', 'Release', 'iohook.node'),
-    //     path.join(__dirname, 'build', 'Release', currentPlatform + '.node')
-    //   );
-    // }
-  }
-} else {
-  proc.execSync('npm remove nan cmake-js');
-  deleteFolderRecursive(path.join(__dirname, 'libuiohook'), true);
-  deleteFolderRecursive(path.join(__dirname, 'build', 'CMakeFiles'));
+function onerror(err) {
+  console.error(err)
 }
 
-function checkLibuiohookSource() {
-  return fs.existsSync(path.join(__dirname, 'libuiohook', 'include', 'uiohook.h'));
+function install(runtime, abi, platform, arch, cb) {
+  const essential = runtime + '-v' + abi + '-' + platform + '-' + arch
+  const pkgVersion = pkg.version
+  const currentPlatform = pkg.name + '-v' + pkgVersion + '-' + essential
+  console.log('Downloading prebuild for platform:', currentPlatform);
+  const downloadUrl = 'https://github.com/vespakoen/iohook/releases/download/v' + pkgVersion + '/' + currentPlatform + '.tar.gz'
+  var reqOpts = { url: downloadUrl }
+  var tempFile = path.join(os.tmpdir(), 'prebuild.tar.gz')
+  var req = get(reqOpts, function (err, res) {
```

Comment:
The "prebuild-install" script actually has some config options for when the end user / developer installing the package is behind a proxy, might be worth adding it in here too.